.env.example

```
@@ -0,0 +1,10 @@
+# ------------------------------
+# Frontend Application Settings
+# ------------------------------
+
+# The port on your host machine to access the website (e.g., http://localhost:8080)
+FRONTEND_PORT=8080
+
+# The full URL to the backend API, as seen from the user's browser.
+# This default works when accessing from the same machine where Docker is running.
+VITE_API_URL=http://localhost:3000/v1/
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)